src/components/flow/routers/dial/DialRouterForm.tsx

```diff
@@ -66,8 +85,41 @@ export default class DialRouterForm extends React.Component<RouterFormProps, Dia
   public renderEdit(): JSX.Element {
     const typeConfig = this.props.typeConfig;
 
+    const advanced: Tab = {
+      name: i18n.t('advanced', 'Advanced'),
+      body: (
+        <>
+          <div className={styles.form}>
+            <p>{i18n.t('forms.dial_limit', 'Dial Limit (sec)')}</p>
+            <TextInputElement
+              name={i18n.t('forms.dial_limit', 'Dial Limit (sec)')}
+              placeholder={'60'}
+              maxLength={2} //max 99s = 1.65min
```

Comment:
ok, so maybe we just enforce a min since you've accounted for max here which seem more than reasonable